### ISSUE TITLE
WTS randomly ignores order param for txns.json

### ISSUE THREAD

**Issue by g4s8** [zold-io/wts.zold.io#308]
To reproduce: run 10 times API query with transactions request: https://wts.zold.io/txns.json?sort=desc
8-9 times WTS will return transactions in correct order (descending), but 1-2 times in ascending order, what is not correct.

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by yegor256** [comment]
@g4s8 are you sure? I can't reproduce it...

**Comment by g4s8** [comment]
@yegor256 yes, I've reproduced it right now (3 requests were OK, 4th inversed)
OK:
![scrot-2019-12-16-10:19:14](https://user-images.githubusercontent.com/1524988/70887056-01237580-1fee-11ea-9d8f-4274eac9c362.png)
Wrong:
![scrot-2019-12-16-10:19:03](https://user-images.githubusercontent.com/1524988/70887057-01237580-1fee-11ea-94c9-dd45cc5d693a.png)

**Comment by yegor256** [comment]
@g4s8 I really can't understand what's going on...
